testsupport/init.go

```diff
@@ -67,6 +69,36 @@ func waitForOperators(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	cl.Create(context.TODO(), &corev1.ServiceAccount{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "e2e_test",
+			Namespace: hostNs}})
+
+	crb := rbacv1.ClusterRoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "e2e_test:admin",
```

Comment:
this doesn't look like a valid name either
```suggestion
			Name: "e2e-test-cluster-admin",
```

Reply:
okay  changing it